src/components/Calculator/Calculator.js

```diff
@@ -74,7 +74,6 @@ export default function Calculator() {
       default:
         return;
     }
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/219302063-569c1ffc-684a-4f79-a43d-ee9a7aa1d3ae.png)

- [ ] Kindly put your deployment link here otherwise add a message here that **This application will be deployed soon...** to make your readme file descriptive and professional :blush: